packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -859,9 +942,18 @@ const cleanup = async (): Promise<void> => {
 
   const filterPredicate = getFilterPredicate(args);
   const accounts = await getAccountsToCleanup();
-
-  await Promise.all(accounts.map((account, i) => cleanupAccount(account, i, filterPredicate)));
+  for(let i = 0 ;i < 5; i ++){
```

Comment:
This works great, but just fyi the package `tiny-async-pool` is already a dependency and does some neat stuff with async generators to make this more succinct and efficient. https://www.npmjs.com/package/tiny-async-pool. It's a nit, but it might shave a few minutes off of this.